torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -22,7 +22,11 @@
 {{micro_gemm.codegen_define(kernel)}}
 
 {%- if x_scale is not none %}
+{%- if is_int8_binary_with_bias %}
+{%- set kernel_args = {"X": X, "x_scale": x_scale, "x_zp": x_zp, "W": W, "w_scale": w_scale, "w_zp": w_zp, "X2": X2, "inp": inp,} %}
+{%- else %}
```

Comment:
Why would the fusion impact the function definition? The fused args should be added automatically by the codegen?

Reply:
Thanks for the comment. After adding `input_indices` to reorder `bias` at front of `x2` in template kernel input nodes, we can remove the extra kernel_args definition.